agent-container/agent-manifest.json

```diff
@@ -0,0 +1,2 @@
+[{"Config":"config.json","RepoTags":["amazon/amazon-ecs-agent:1.60.0"],"Layers":["rootfs/layer.tar"]}]
```

Comment:
this one should probably be generated at build-time too?

Reply:
I'll update the repoTags to use the VERSION.
These are generated by Docker otherwise -- if we don't have Docker, we need to create/provide these config files manually.  

Bottlerocket uses static configs to build its pause container: https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/ecs-agent/pause-manifest.json